README.md

```diff
@@ -56,6 +45,28 @@ See the [documentation](http://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/#u
 }
 ```
 
+### Create a `Dockerfile` in your Node.js app project
+Assuming that `pm2.json` and `package.json` are in the same folder of the `Dockerfile` and that the `src` folder contains your source code.
+
+```dockerfile
+FROM keymetrics/pm2-docker-alpine:latest
+
+# Bundle APP files
+COPY src src/
+COPY package.json .
+COPY pm2.json .
```

Comment:
And by the way i think we should just redirect user to [here](http://pm2.keymetrics.io/docs/usage/application-declaration/#generate-configuration) because it explain all the format user can use, you could just assume they will name the file `pm2.json`